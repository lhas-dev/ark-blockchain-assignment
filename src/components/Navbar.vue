<template>
  <nav class="flex items-center justify-center flex-wrap bg-red-600 p-5">
    <div class="flex items-center flex-shrink-0 text-white mr-6">
      <img class="logo" src="../assets/ark-logo.svg" />
      <router-link to="/" class="font-semibold text-xl tracking-tight pl-2">Paper Wallet</router-link>
    </div>
  </nav>
</template>

<script>
export default {};
</script>

<style scoped>
.logo {
  height: 32px;
}
</style>
