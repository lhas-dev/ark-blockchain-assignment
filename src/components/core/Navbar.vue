<template>
  <nav class="flex items-center justify-between flex-wrap bg-red-600 p-5">
    <div class="flex items-center flex-shrink-0 text-white mr-6">
      <img src="./../../assets/ark-logo.svg" />
      <span class="font-semibold text-xl tracking-tight pl-2">
        Paper Wallet</span
      >
    </div>
    <div class="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
      <div class="text-sm lg:flex-grow"></div>
      <div>
        <a
          href="#"
          class="inline-block text-sm px-4 py-2 leading-none border rounded text-white border-white hover:border-transparent hover:text-teal-500 hover:bg-white mt-4 lg:mt-0"
          >Sign In</a
        >
      </div>
    </div>
  </nav>
</template>

<script>
export default {};
</script>

<style>
img {
  height: 32px;
}
</style>
