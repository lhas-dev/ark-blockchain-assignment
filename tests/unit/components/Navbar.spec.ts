import { shallowMount } from "@vue/test-utils";
import Navbar from "@/components/Navbar.vue";

describe("Navbar.vue", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(Navbar, {
      stubs: ["router-link", "router-view"]
    });
    expect(wrapper).toMatchSnapshot();
  });
});
