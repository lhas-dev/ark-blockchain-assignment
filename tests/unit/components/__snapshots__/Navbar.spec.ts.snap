// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar.vue renders correctly 1`] = `
<nav class="flex items-center justify-center flex-wrap bg-red-600 p-5">
  <div class="flex items-center flex-shrink-0 text-white mr-6"><img src="../assets/ark-logo.svg" class="logo">
    <router-link-stub to="/" class="font-semibold text-xl tracking-tight pl-2">Paper Wallet</router-link-stub>
  </div>
</nav>
`;
